python/dgl/graphbolt/data_format.py

```diff
@@ -11,14 +11,26 @@ class LinkPredictionEdgeFormat(Enum):
     in link prediction:
 
     Attributes:
+    INDEPENDENT: Represents the 'independent' format where data is structured
+    as triples `[u, v, label]` indicating the source and destination nodes of
+    an edge, with a label (0 or 1) denoting it as negative or positive.
+
     CONDITIONED: Represents the 'conditioned' format where data is
     structured as quadruples `[u, v, [negative heads], [negative tails]]`
```

Comment:
Worth to callout the [negative heads], [negative tails] should have the same length.